kyaml/openapi/kubernetesapi/openapiinfo.go

```diff
@@ -6,13 +6,13 @@
 package kubernetesapi
```

Comment:
This and `kyaml/openapi/kubernetesapi/v1212/swagger.go` are auto-generated files. What caused the diff?

Reply:
`hack/kyaml-pre-commit.sh` -- the diff that caused is in its own commit on this PR

Reply:
ack